apps/deploy-web/tests/deploy-hello-world.spec.ts

```diff
@@ -0,0 +1,27 @@
+import { DeployHelloWorldPage } from "./pages/DeployHelloWorldPage";
+import { test } from "./utils/fixture";
+import { setupLeap } from "./utils/wallet";
+
+// test.describe.configure({ mode: "serial" });
+
+test.only("deploy hello world", async ({ extPage: page, context }) => {
+  await setupLeap(context, page);
+
+  const customTemplatePage = new DeployHelloWorldPage(context, page, "new-deployment", "hello-world-card");
+
+  // Create deployment
+  await customTemplatePage.gotoInteractive();
```

Comment:
This could probably go in the `DeployHelloWorldPage` class under a single function `deploy()` wdyt? @ygrishajev

Reply:
It can be done for sure. The comments and grouping of the method calls below also asks for more methods)) e.g.

```typescript
async validateLeaseAndClose() {
   await customTemplatePage.validateLease();
   await customTemplatePage.closeDeploymentDetail();
   await customTemplatePage.signTransaction();
}
```